docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md

```diff
@@ -0,0 +1,256 @@
+    ```
+
+    :::
+
+    The Palette CLI prompts you for information regarding the OVA you want to import.
+
+    | **Parameter**                               | **Description**                                                                                                                                                                                   | **Values**                        |
+    | ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------- |
+    | **OVA Path**                                | The path to the image your have uploaded to your VM. The path for the example provided is `/root/bitnami-wordpress-6.2.2-r1-debian-11-amd64.ova/`.                                                |                                   |
+    | **Container Disk Upload Method**            | Indicate whether to upload the image directly to the target cluster as a `DataVolume` or build and push a Docker image. You will need to provide an existing image registry if you select Docker. | `DataVolume` / `Docker Image`     |
+    | **Kubeconfig Path**                         | The path to the kubeconfig file you have uploaded to your VM.                                                                                                                                     |                                   |
+    | **DataVolume Namespace**                    | The namespace to create your `DataVolume`, if you selected this option previously.                                                                                                                |                                   |
+    | **DataVolume Name**                         | The name of your `DataVolume`.                                                                                                                                                                    |                                   |
+    | **Overhead Percentage for DataVolume Size** | Set an overhead percentage for your `DataVolume` compared to the OVA specification. This parameter is optional and can be skipped with the value `-1`.                                            |                                   |
+    | **Access Mode for the PVC**                 | Set the access mode for your `DataVolume`. Ensure that your configured CSI supports your selection.                                                                                               | `ReadWriteMany` / `ReadWriteOnce` |
+    | **Create a PVC with VolumeMode=Block**      | Indicate whether to set `Block` volume mode on the `DataVolume`.                                                                                                                                  | `y` / `N`                         |
```

Comment:
If yes is chosen for this prompt, the containerd config patch documented elsewhere must be in place!

Reply:
I have added the same containerd config patch from the migration here.